google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java

```diff
@@ -0,0 +1,171 @@
+package com.google.cloud.examples.securitycenter.snippets;
+
+import com.google.cloud.securitycenter.v1beta1.ListAssetsRequest;
+import com.google.cloud.securitycenter.v1beta1.ListAssetsResponse.ListAssetsResult;
+import com.google.cloud.securitycenter.v1beta1.OrganizationName;
+import com.google.cloud.securitycenter.v1beta1.SecurityCenterClient;
+import com.google.cloud.securitycenter.v1beta1.SecurityCenterClient.ListAssetsPagedResponse;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import org.threeten.bp.Duration;
+import org.threeten.bp.Instant;
+import org.threeten.bp.LocalDateTime;
+import org.threeten.bp.ZoneId;
+import org.threeten.bp.temporal.ChronoUnit;
+
+/** Snippets for how to work with Assets in Cloud Security Command Center. */
+public class AssetSnippets {
+
+  private final SecurityCenterClient securityCenterClient;
+  private final OrganizationName organizationName;
+
+  /** Filter that returns all projects in the organization */
+  // [START asset_resource_project_filter]
+  public static final String PROJECT_ASSET_FILTERS =
+      "security_center_properties.resource_type=\"google.cloud.resourcemanager.Project\"";
+  // [END asset_resource_project_filter]
+
+  /**
+   * Create a new AssetSnippets object.
+   *
+   * @param client The client to use for contacting the service.
+   * @param organizationId The organization ID (this should be a numeric value, not the display name
+   *     of the organization).
+   */
+  public AssetSnippets(SecurityCenterClient client, String organizationId) {
+    this.securityCenterClient = client;
+    // [START name_from_id]
+    this.organizationName = OrganizationName.of(organizationId);
+    // [END name_from_id]
+  }
+
+  /**
+   * Lists assets for an organization given meeting <code>filter</code> as of a specific instant in
+   * time.
+   *
+   * @param filter The filter that assets must meet (e.g. {@link #PROJECT_ASSET_FILTERS}). If null,
+   *     all assets in the organization are returned.
+   * @param asOf The instant in time to query for. If null, current time is assumed
+   */
+
+  // [TARGET list_assets]
+  // [VARIABLE "filter"]
+  // [VARIABLE "as_of"]
+  // [START list_assets]
+  public ImmutableList<ListAssetsResult> listAssets(String filter, Instant asOf) {
+    ListAssetsRequest.Builder request =
+        ListAssetsRequest.newBuilder().setParent(organizationName.toString());
```

Comment:
Please add commented variables with a small description of what it's for?
Similar to: https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java#L1459-L1460

Reply:
I think I documented what you asked for